uint8 agent_number
std_msgs/Bool check
